packages/app-store/zohocrm/_metadata.ts

```diff
@@ -0,0 +1,10 @@
+import type { AppMeta } from "@calcom/types/App";
+
+import config from "./config.json";
+
+export const metadata = {
+  dirName: "zohocrm",
```

Comment:
You don't need to add `dirName` in config.json, the default is the name of the directory itself.